magefiles/magefile.go

```diff
@@ -347,6 +347,28 @@ func RunE2ETests() error {
 		return nil
 	}
 
+	if openshiftJobSpec.Refs.Repo == "infra-deployments" {
+
+		rctx.RepoName = openshiftJobSpec.Refs.Repo
+		rctx.JobName = jobName
+		rctx.JobType = jobType
+
+		files, err := getChangedFiles()
```

Comment:
I think this won't work. We need to get list of changed files from within the infra-deployments directory, but this function lists files from the current (e2e-tests) directory.

I'd suggest to update the `getChangedFiles` func to accept the parameter for the location of the directory it should run `git diff` against.
@Dannyb48 wdyt?

Reply:
agree, 
it didn't work on the **infra-deployments** , so I've fixed the function 
to get the repo and to check the absolute path of **infra-deployments** 
tested and working fine , see examples in description of PR